### ISSUE TITLE
Fractions are broken

### ISSUE THREAD

**Issue by mB-PiBox** [nikolaeu/numi#179]
![image](https://user-images.githubusercontent.com/14003870/93639481-ab398f00-f9be-11ea-9dbf-7c4200a1e5e1.png)

Fractions don't seem to properly convert or get detected. Perhaps I'm entering them incorrectly, and the / is acting as a divide? But even then the calculation total is not correct.

I installed NUMI through Setapp, so that I could make quick measurement conversion when using Fusion360 instead of having to have the calculator open, and write down measurements I need to refer to later.. but NUMI is not working as expected when it comes to fractions and/or conversion. 

It will convert whole number, or ones with decimals from inch to mm, but not when a fraction is used.

**Comment by es183923** [comment]
it detects `4 1/2 + 9` as `41/2 + 9`, which is equal to `29.5`.  Same for the other fractions.

**Comment by Benmidi** [comment]
Hey @mB-PiBox, you just need a bit more structure,
`(4 and 1/4) inches to mm` will do the trick!

<img width="520" alt="Screen Shot 2020-11-24 at 9 47 21 PM" src="https://user-images.githubusercontent.com/813586/100188384-ec5a8d80-2e9e-11eb-8a02-550cd48acf92.png">

**Comment by mB-PiBox** [comment]
> Hey @mB-PiBox, you just need a bit more structure,
> `(4 and 1/4) inches to mm` will do the trick!
> 
> <img alt="Screen Shot 2020-11-24 at 9 47 21 PM" width="520" src="https://user-images.githubusercontent.com/813586/100188384-ec5a8d80-2e9e-11eb-8a02-550cd48acf92.png">

No, I think the app should proper detect Fractions. Like a typical calculator would. Shouldn't have to type out extra characters.
